cedar-drt/README.md

```diff
@@ -5,21 +5,20 @@ See the README in the toplevel directory `..` for instructions on how to run.
 
 ## Available fuzz targets
 
-The table below lists all available fuzz targets, including which component of the code they test and whether they perform property-based testing of the Rust code (PBT) or differential testing of the Rust code against the Dafny spec (DRT).
+The table below lists all available fuzz targets, including which component of the code they test and whether they perform property-based testing of the Rust code (PBT) or differential testing of the Rust code against the Lean spec (DRT).
 
 | Name | Component(s) tested | Type | Description |
 | ----------- | ----------- | ----------- | ----------- |
-| [`abac-type-directed`](fuzz/fuzz_targets/abac-type-directed.rs) | Evaluator | DRT | Diff test evaluator on ABAC policies using (mostly) well-typed inputs |
-| [`abac`](fuzz/fuzz_targets/abac.rs) | Evaluator | DRT | Diff test evaluator on ABAC policies |
+| [`abac-type-directed`](fuzz/fuzz_targets/abac-type-directed.rs) | Authorizer | DRT | Diff test authorizer on ABAC policies using (mostly) well-typed inputs |
+| [`abac`](fuzz/fuzz_targets/abac.rs) | Authorizer | DRT | Diff test evaluator on ABAC policies |
```

Comment:
```suggestion
| [`abac`](fuzz/fuzz_targets/abac.rs) | Authorizer | DRT | Diff test authorizer on ABAC policies |
```

Reply:
IMO, we don't actually test much of the authorizer using this target because we only generate one policy per run.

Reply:
Yeah I was thinking this too as I was making the change, but it seems closer to authorization than the `eval-type-directed` target so I'm inclined to stick with "authorizer".